pkg/monitoring/observability/rules/alerts/cluster_alerts.go

```diff
@@ -116,5 +116,21 @@ func clusterAlerts() []promv1.Rule {
 				"operator_health_impact": "none",
 			},
 		},
+		{
+			Alert: "DuplicateWaspAgentDSDetected",
+			Expr: intstr.FromString(
+				`count(kube_daemonset_metadata_generation{namespace="wasp"}) > 0
+					and kubevirt_hco_memory_overcommit_percentage > 100
+			`),
+			For: ptr.To[promv1.Duration]("1m"),
+			Annotations: map[string]string{
+				"summary":     "Duplicate wasp-agent deployment detected",
+				"description": "In the current situation two wasp-agent deployments exist in the same time, the deployment in the wasp namespace should be deleted",
+			},
+			Labels: map[string]string{
+				"severity":               "warning",
+				"operator_health_impact": "none",
```

Comment:
You dont believe this can impact the operator functionality ?

Reply:
@sradco AFAIK it shouldn't, Ive tested this locally as well. The duplication of the components won't affect HCO ability to deploy because they are located on different namespaces and also the names are different.

Reply:
Its not about the ability to deploy, but rather a matter of actual impact on the workloads.
Which ds will win if they are both deployed?